src/app/docs/utils.js

```diff
@@ -0,0 +1,36 @@
+const _ = require('underscore');
+export function getReferences(project, model) {
```

Comment:
We've historically leveraged services for things like this, but I'm increasingly super into the idea of just using a `require` instead of going through Angular's dependency injection flow.

Can you just rename this from `utils.js` to something more specific like `dag_utils.js` instead?

Reply:
Can do!